### PR TITLE
ipi-deprovision: fix failure report

### PR DESCRIPTION
**Pull request by bbguimaraes**
Prevents failures such as:

```
ls: cannot access '/logs/artifacts/deprovision/*/failure': No such file or directory
```

https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ipi-deprovision/1336292322861649920

### CODE REVIEW

File: cmd/ipi-deprovision/ipi-deprovision.sh [hunk 1]
```diff
@@ -89,11 +90,11 @@ if [[ "${#buckets[@]}" -gt 0 ]]; then
   timeout 30m gsutil -m rm -r "${buckets[@]}"
 fi
 
-FAILED="$(ls "${ARTIFACTS}"/deprovision/*/failure)"
+FAILED="$(find ${clusters} -name failure -printf '%H\n')"
```
Comment (alvaroaleman) [inline cmd/ipi-deprovision/ipi-deprovision.sh:93]: What does the `%H` do? When I tried this out it always printed a literal dot newline

File: cmd/ipi-deprovision/ipi-deprovision.sh [hunk 2]
```diff
@@ -89,11 +90,11 @@ if [[ "${#buckets[@]}" -gt 0 ]]; then
...
 if [[ -n "${FAILED}" ]]; then
   echo "Deprovision failed on the following clusters:"
-  echo "${FAILED}"
+  sed 's,.*/,,' <<< $FAILED
```
Comment (alvaroaleman) [inline cmd/ipi-deprovision/ipi-deprovision.sh:96]: This isn't incorrect, but using the `basename` CLI makes this much easier to undestand ;)

File: cmd/ipi-deprovision/ipi-deprovision.sh (hunk 1 above)
Comment (bbguimaraes) [reply cmd/ipi-deprovision/ipi-deprovision.sh]:
```
%H     Starting-point under which file was found.
```

File: cmd/ipi-deprovision/ipi-deprovision.sh (hunk 2 above)
Comment (bbguimaraes) [reply cmd/ipi-deprovision/ipi-deprovision.sh]: Changed to `xargs --max-args 1 basename`.

File: cmd/ipi-deprovision/ipi-deprovision.sh (hunk 1 above)
Comment (alvaroaleman) [reply cmd/ipi-deprovision/ipi-deprovision.sh]: Doesn't that mean this will always print the value of the `clusters` variable?

Comment (bbguimaraes) [reply cmd/ipi-deprovision/ipi-deprovision.sh]: `$clusters` is a list of directories (see line 72).  `find` will print the directory that originated the search that found each file named `failure`:

```
$ tree
.
└── deprovision
    ├── a
    │   └── failure
    ├── b
    ├── c
    │   └── failure
    └── d

5 directories, 2 files
$ find deprovision/* -name failure -printf '%H\n'
deprovision/a
deprovision/c
$ ARTIFACTS=$PWD ipi-deprovision/ipi-deprovision.sh
Deprovision failed on the following clusters:
a
c
```

**Pull request merged**